loki/logpipeline-custom.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: telemetry.kyma-project.io/v1alpha1
```

Comment:
is it relevant to have "custom" in the file name? Will make copy/pasting more cumbersome